compiler/src/iree/compiler/Pipelines/Pipelines.cpp

```diff
@@ -145,16 +145,31 @@ void buildIREEVMTransformPassPipeline(
       break;
   }
 
+  IREE::HAL::PipelinePhase halCompileTo;
+  switch (compileTo) {
+    default:
+      halCompileTo = IREE::HAL::PipelinePhase::End;
+      break;
+    case IREEVMPipelinePhase::ExecutableSources:
+      halCompileTo = IREE::HAL::PipelinePhase::ExecutableSources;
+      break;
+    case IREEVMPipelinePhase::ExecutableTargets:
+      halCompileTo = IREE::HAL::PipelinePhase::ExecutableTargets;
+      break;
+  }
+
   IREE_TRACE_ADD_BEGIN_FRAME_PASS(passManager, "HAL");
   switch (schedulingOptions.executionModel) {
     case SchedulingOptions::ExecutionModel::HostOnly:
       // No HAL required.
       break;
     default:
     case SchedulingOptions::ExecutionModel::AsyncInternal:
-    case SchedulingOptions::ExecutionModel::AsyncExternal:
-      IREE::HAL::buildHALTransformPassPipeline(passManager, executableOptions);
+    case SchedulingOptions::ExecutionModel::AsyncExternal: {
+      IREE::HAL::buildHALTransformPassPipeline(passManager, executableOptions,
+                                               halCompileTo);
       break;
+    }
```

Comment:
Why add braces around just this case? Want to also add braces around InlineStatic and InlineDynamic?

Reply:
Ah, I originally had more code in here and that needed it, but moved the code up. Will fix.